### PR TITLE
Fix 37

### PR DESCRIPTION
**Pull request by javpintos**
Uppercase in team.html

### CODE REVIEW

File: team.html [hunk 1]
```diff
@@ -115,7 +115,7 @@ <h2 class ="custom-size">Anubhav Singhal</h2>
             <div class="box">
                 <div class="profile-img"><img class="img-fluid" src="images/team/6.jpeg" style="border-radius: 50%;"></div>
                 <h2>Dheeraj Kotwani</h2>
-                <p> I am an Android Developer and a competitve programmer. </p>
+                <p> I am an Android Developer and a competitve Programmer. </p>
```
Comment (Shruti3004) [inline team.html:118]: Make C also capitl then

**Comment by eamanu** [comment]
ping @javpintos also please, fix the conflict ;)

**Comment by harikesh409** [comment]
@javpintos what is the progress on the requested changes it's been a month if you are not working on it we will assign it to someone else.

**Comment by mrunal512** [comment]
@harikesh409 please assign me this request !
I will start working right away

**Comment by mrunal512** [comment]
@harikesh409 please assign me this issue 
I will start working right away

**Comment by mrunal512** [comment]
@Shruti3004 please assign me this issue

**Comment by harikesh409** [comment]
> @harikesh409 please assign me this issue
> I will start working right away

Go ahead

**Comment by harikesh409** [comment]
Resolved in #511

**Pull request closed without merging**